scripts/nginx/prowlarr.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Nginx conf for *Arr
+# Flying sausages 2020
+# Refactored by Bakerboy448 2021
+master=$(cut -d: -f1 < /root/.master.info)
+app_name="prowlarr"
+if ! PROWLARR_OWNER="$(swizdb get $app_name/owner)"; then
+    PROWLARR_OWNER=$(_get_master_username)
+else
+    PROWLARR_OWNER="$(swizdb get $app_name/owner)"
+fi
+app_port="9696"
+app_sslport="6969"
+user="$PROWLARR_OWNER"
+app_servicefile="${app_name}".service
+app_configdir="/home/$user/.config/${app_name^}"
+app_baseurl=$app_name
+
+cat > /etc/nginx/apps/$app_name.conf << PROWLARR
+location /$app_baseurl {
+  proxy_pass        http://127.0.0.1:$app_port/;
+  proxy_set_header Host \$proxy_host;
+  proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+  proxy_set_header X-Forwarded-Proto \$scheme;
+  proxy_redirect off;
+  auth_basic "What's the password?";
+  auth_basic_user_file /etc/htpasswd.d/htpasswd.${master};
+
+  proxy_http_version 1.1;
+  proxy_set_header Upgrade \$http_upgrade;
+  proxy_set_header Connection \$http_connection;
+  # Allow the App API
+  location /$app_baseurl/api { auth_request off;
+    proxy_pass http://127.0.0.1:$app_port/$app_baseurl/api;
+  }
+  # Allow Content
+  location /$app_baseurl/Content { auth_request off;
+    proxy_pass http://127.0.0.1:$app_port/$app_baseurl/Content;
+  }
+  # Allow Indexers
+  location ~ /$app_baseurl/[0-9]+/api { auth_request off
+    proxy_pass       http://127.0.0.1:9696/$app_baseurl/$1/api;
```

Comment:
what's `$1`? this is an args var, and it's also unescaped so it will expand, which is most likely therefore going to expand to nothing

Reply:
if this is to substitute whatever `[0-9]` is, then it needs to be escaped
```suggestion
    proxy_pass       http://127.0.0.1:9696/$app_baseurl/\$1/api;
```